controllers/suite_test.go

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
Is this file necessary? Or why is it commented out?

Reply:
This file was generated by `operator-sdk` . I kept it because I thought we might need to move e2e-tests in this file in the future. Currently we are using `operator-framework` . Let me know if it needs to be removed.